flake.nix

```diff
@@ -25,6 +25,11 @@
       inputs.nixpkgs-25_05.follows = "nixpkgs";
       inputs.flake-compat.follows = "flake-compat";
     };
+    tahoe-lafs = {
+      url = "github:tahoe-lafs/tahoe-lafs";
+      inputs.flake-compat.follows = "flake-compat";
+      inputs.nixpkgs-24_11.follows = "nixpkgs-24_11";
```

Comment:
It seem like the current head of this flake does not follow yet nixpkgs-25.05!
But I doubt we will be working on this update soon.
So I guess we are already pulling a lot of nixpkgs-24.11 to run tahoe-lafs on the testgrid today, despite this server being running nixos 25.05 for some time now.